### PR TITLE
add tests to increase coverage.

### PR DESCRIPTION
**Pull request by devinleighsmith**
fix legend markers.
fix deleting agencies with users.

### CODE REVIEW

File: backend/dal/Services/Admin/Concrete/AgencyService.cs [hunk 1]
```diff
@@ -207,7 +207,7 @@ public override void Remove(Agency agency)
         {
             agency.ThrowIfNull(nameof(agency));
 
-            var original = this.Context.Agencies.Find(agency.Id) ?? throw new KeyNotFoundException();
+            var original = this.Context.Agencies.Include(a => a.Users).FirstOrDefault(a => a.Id == agency.Id) ?? throw new KeyNotFoundException();
```
Comment (devinleighsmith) [inline backend/dal/Services/Admin/Concrete/AgencyService.cs:210]: not including the useragencies causes the cascade to fail.

File: frontend/src/components/maps/leaflet/Legend/Legend.tsx [hunk 2]
```diff
@@ -6,23 +6,23 @@ export const Legend = () => {
   const keys = React.useMemo(() => {
     return [
       {
-        pin: require('assets/images/pins/land-reg.png'),
+        pin: require('assets/images/pins/land-reg.png').default,
```
Comment (devinleighsmith) [inline frontend/src/components/maps/leaflet/Legend/Legend.tsx:9]: react scripts 4 loads requires images as modules.

**Comment by devinleighsmith** [comment]
also a lot of the test warnings have been re-introduced, will fix that in a follow-up.

File: backend/dal/Services/Admin/Concrete/AgencyService.cs (hunk 1 above)
Comment (Fosol) [reply backend/dal/Services/Admin/Concrete/AgencyService.cs:210]: The downside to this is `include(...)` statements are expensive operations.

**Review by Fosol — APPROVED** [review]

**Review by goemen — APPROVED** [review]

File: backend/dal/Services/Admin/Concrete/AgencyService.cs (hunk 1 above)
Comment (devinleighsmith) [reply backend/dal/Services/Admin/Concrete/AgencyService.cs:210]: this should be fine since it is finding by an indexed id.

**Review by jdtoombs — APPROVED** [review]

**Pull request merged**